ui/app/selectors/swap.js

```diff
@@ -1,12 +1,6 @@
 import abi from 'human-standard-token-abi'
-import {
-  accountsWithSwapEtherInfoSelector,
-  getAddressBook,
-  getSelectedAccount,
-  getTargetAccount,
-  getAveragePriceEstimateInHexWEI,
-} from '.'
-import { calcGasTotal } from '../pages/swap/swap.utils'
+import { accountsWithSwapEtherInfoSelector, getAddressBook, getSelectedAccount, getTargetAccount } from '.'
```

Comment:
Do we need the AddressBook? I think this is code inherited from `send` which we might not need.

Reply:
Indeed, will remove those stuff. 👍